service/src/main/kotlin/fi/espoo/evaka/daycare/CareArea.kt

```diff
@@ -64,6 +68,12 @@ data class Daycare(
     val operationDays: Set<Int>
 )
 
+@PropagateNull("finance_decision_handler_id")
+data class financeDecisionHandler(
+    @Nested("")
+    val employee: Employee
+)
```

Comment:
Instead of having this nested structure that pulls data that is ultimately not needed, just change this into a simple data class. Something like this 
```
data class FinanceDecisionHandler(val id: UUID, val firstName: String, val lastName: String)
```